### PR TITLE
wrote initial implementation

### LINKED ISSUE #1: setup initial implementation of spectral config

### PR DESCRIPTION
**Pull request by lukebrdn**
Resolves #1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,36 +1,42 @@
...
 ```
-import * as insertName from '@checkdigit/< Insert name >';
 
-const someVariable = insertName();
+Also add the following to your `package.json` to run Spectral:
 
+```jsonc
+"scripts": {
+  // ...
+  "lint:spec": "spectral lint src/**/swagger.yml",
```
Comment (ramaghanta) [inline README.md:25]: should this be `lint:swagger` or `lint:spectral` or `lint:openapi`? `lint:spec` implies linting spec files.

File: package.json [hunk 2]
```diff
@@ -1,55 +1,57 @@
...
   "version": "0.0.1",
-  "description": "A short description of the intended use of the library",
-  "typings": "./dist/index.d.ts",
-  "main": "./dist/index.js",
+  "description": "Check Digit Spectral Config",
+  "main": "ruleset.json",
   "prettier": "@checkdigit/prettier-config",
   "engines": {
     "node": ">=12.18",
     "npm": ">=6.14"
```
Comment (ramaghanta) [inline package.json:9]: At least node 16 and npm 8?

File: package.json [hunk 3]
```diff
@@ -1,55 +1,57 @@
...
+    "ci:style": "npm run prettier"
+  },
+  "eslintConfig": {
+    "extends": [
+      "@checkdigit/eslint-config"
+    ]
+  },
+  "jest": {
+    "preset": "@checkdigit/jest-config",
+    "testTimeout": 100000
```
Comment (ramaghanta) [inline package.json:55]: `testTimeout` is probably not required

**Review by ramaghanta — CHANGES_REQUESTED** [review]
assuming the coverage check was failing due to missing stuff in main? Also, missing a label.

File: README.md (hunk 1 above)
Comment (lukebrdn) [reply README.md]: yep, that makes sense. changing to lint:openapi.

**Comment by lukebrdn** [comment]
> assuming the coverage check was failing due to missing stuff in main? Also, missing a label.
was failing because I didn't include jest-config, added it. now it should fail for the expected reason.

**Review by ramaghanta — APPROVED** [review]
lgtm

**Review by carlansley — APPROVED** [review]

**Review by bobwieler — APPROVED** [review]
lgtm

**Pull request merged**